src/main/java/com/untamedears/humbug/Humbug.java

```diff
@@ -2383,53 +2384,48 @@ public void run() {
   public final static int pearlId = 368;
   public final static MinecraftKey pearlKey = new MinecraftKey("ender_pearl");
 
-  @SuppressWarnings({ "rawtypes", "unchecked" })
+  @SuppressWarnings("unchecked")
   @BahHumbug(opt="ender_pearl_gravity", type=OptType.Double, def="0.060000")
-  private void hookEnderPearls() {
+  public void hookEnderPearls() {
```

Comment:
This is very good, thank you.

One thing: Can you add a catch against the gravity config, where if it == 0.0, we do not hook ender pearls?